.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Test collection
+on:  # yamllint disable-line rule:truthy
```

Comment:
Can you also add the same concurrency bits from this PR: https://github.com/ansible-collections/cisco.nxos/pull/516/files#

It will prevent multiple github action runs for the same PR when pushes to it are back-to-back